service/routes.py

```diff
@@ -26,12 +26,27 @@ def index():
     """ Root URL response """
     return ("<p>The inventory service keeps track of how many of each product we have in our warehouse</p>",
             status.HTTP_200_OK)
-
+    
 
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+######################################################################
+# DELETE A INVENTORY ITEM
+######################################################################
+@app.route("/inventory/<int:product_id>/<string:condition>", methods=["DELETE"])
+def delete_inventory(product_id, condition):
+    '''This endpoint will delete a product with the specified id and condition'''
+    app.logger.info("Request to delete a product with product_id %s and condition %s", product_id, condition)
+    product = Inventory.query.filter_by(product_id=product_id, condition=condition).first()
+    if product:
+        product.delete()
+        # db.session.commit()
+        app.logger.info("Product with product_id %s and condition %s deleted.", product_id, condition)
+        return ("", status.HTTP_204_NO_CONTENT)
+    return ("", status.HTTP_404_NOT_FOUND)
```

Comment:
The default response code should be `204 no content`, even if the item to be deleted was not present in the DB

Reply:
@MarvinTellis Should I go for 204 or 404. I am bit confused here as per the API contract which mentions the following. 

"Given a request to delete a non-existent item from the inventory,
When the API is called with the appropriate parameters,
Then the API should respond with a status code of 404 (Not Found).
And the response body should contain an error message indicating that the specified shopcart item was not found.
The format should look like this:

{
    <error> : 404
    <error_code> : Not Found
} "

Reply:
Okay, so we are going for 204 ? Should I make the change ?

Reply:
Yes. I have already updated the story on Kanban board. You can make the changes in code

Reply:
Done. Thank you